src/main/java/com/hazelcast/eureka/one/EurekaOneDiscoveryStrategy.java

```diff
@@ -239,7 +278,7 @@ void verifyEurekaRegistration() {
 
         @Override
         public boolean shouldRegisterWithEureka() {
-            return !clientMode && selfRegistration;
```

Comment:
We don't consider `clientMode` anymore. Is that correct behavior? cc: @noctarius

Reply:
From my perspective a client should never register itself. Good catch.

Reply:
@lazerion any idea?

Reply:
Client is not registering itself, please see builder and added strategy approach for registering.

Reply:
You may also check test cases in `EurekaOneDiscoveryStrategyClientTest`

Reply:
Ok. I see now. Thx!